scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -101,8 +109,11 @@ namespace <%= namespace %> {
                     return;
                 }
 
-                UpdateCheckout(response.checkoutCreate().checkout());
-                CheckoutPoll(callback);
+                if (UpdateState(response.checkoutCreate().checkout(), response.checkoutCreate().userErrors())) {
```

Comment:
No those are GraphQL errors. This is basically syntax errors when writing the GraphQL query.